src/NodaTime/Text/LocalTimePattern.cs

```diff
@@ -50,6 +50,32 @@ public sealed class LocalTimePattern : IPattern<LocalTime>
         /// <value>An invariant local time pattern which is ISO-8601 compatible, with no sub-second precision.</value>
         public static LocalTimePattern GeneralIso => Patterns.GeneralIsoPatternImpl;
 
+        /// <summary>
+        /// Gets an invariant local time pattern which is ISO-8601 compatible, with precision of just minutes.
+        /// This corresponds to the text pattern "HH':'mm':'ss".
+        /// </summary>
+        /// <value>An invariant local time pattern which is ISO-8601 compatible, with no sub-minute precision.</value>
+        public static LocalTimePattern HourMinuteIso => Patterns.HourMinuteIsoPatternImpl;
+
+        /// <summary>
+        /// Gets an invariant local time pattern which is ISO-8601 compatible, with a precision of just hours.
+        /// This corresponds to the text pattern "HH".
+        /// </summary>
+        /// <value>An invariant local time pattern which is ISO-8601 compatible, with no sub-hour precision.</value>
+        public static LocalTimePattern HourIso => Patterns.HourIsoPatternImpl;
+
+        /// <summary>
+        /// Gets an invariant local time pattern which can parse any ISO-8601 compatible value, (just hours;
```

Comment:
Not quite sure about "... any ISO-8601 compatible value, (just ...)." Are there ISO-8601 time formats that this can't parse? (If so, perhaps "any ... value that ..."?) If not, then I'm not sure what the parenthetical is trying to say.

Reply:
The parenthetical part is meant to emphasize where this is different to the other patterns. Maybe they should be in a new sentence instead of in parentheses? How about:

> Gets an invariant local time pattern which can parse any ISO-8601 compatible value, regardless of precision. Valid values include "just hours", "hours and minutes", "hours, minutes and seconds", and extended values with fractions of seconds (as far as nanoseconds).

That said, possibly "any value" is overstating it, given that we *do* require colons. Perhaps "any value in extended format" (as opposed to basic format, which could be 05, 0510, 051020).

Reply:
Yes, two sentences is much clearer, thanks!

We don't really do much with the basic format (and most of our patterns don't mention basic or extented [e.g. `GeneralIso` above]). How about "can parse any ISO-8601 compatible value (in extended format, that is, with separators), regardless of..." ?